unlock-app/package.json

```diff
@@ -91,5 +92,8 @@
     "not dead",
     "not ie <= 11",
     "not op_mini all"
-  ]
+  ],
+  "devDependencies": {
```

Comment:
I don't think this is wholly accurate. This article explains the reasoning: https://silvenon.com/blog/dependencies-or-devdependencies

Basically, jest, jest-mock-axios, and all the testing things are not needed in order to actually build unlock-app or to execute its code, so they should go into devDependencies. If you feel strongly after reading this, I'll happily move it. I suspect that the only disadvantage will be build time, since webpack prunes dead code now, but that could be significant as the app gets larger.

Reply:
This is great feedback. I opened an issue so we start thinking about migrating all other `dependencies` which are not required to build to `devDependencies `